iij/ngx_http_secure_token_iijpta.c

```diff
@@ -0,0 +1,191 @@
+#include "ngx_http_secure_token_iijpta.h"
+#include "../ngx_http_secure_token_filter_module.h"
+#include "../ngx_http_secure_token_utils.h"
+
+#include <time.h>
+#include <openssl/hmac.h>
+#include <openssl/evp.h>
+
+// constants
+#define CRC32_SIZE    4
+#define DEADLINE_SIZE 8
+#define PATH_LIMIT    1024
+	
+// typedefs
+typedef struct {
+	ngx_str_t key;
+	ngx_str_t iv;
+	ngx_str_t path;
+	time_t    timelimit;
+} ngx_secure_token_iijpta_token_t;
+
+// globals
+static ngx_command_t ngx_http_secure_token_iijpta_cmds[] = {
+	{ ngx_string("key"),
+	NGX_CONF_TAKE1,
+	ngx_http_secure_token_conf_set_hex_str_slot,
+	0,
+	offsetof(ngx_secure_token_iijpta_token_t, key),
+	NULL },
+	
+	{ ngx_string("iv"),
+	NGX_CONF_TAKE1,
+	ngx_http_secure_token_conf_set_hex_str_slot,
+	0,
+	offsetof(ngx_secure_token_iijpta_token_t, iv),
+	NULL },
+
+	{ ngx_string("path"),
+	NGX_CONF_TAKE1,
+	ngx_conf_set_str_slot,
```

Comment:
better change to complex value, in order to enable the user to use nginx variables when defining it. see the acl parameter in Akamai. would also be a good idea to rename it to acl, for consistency (it has exactly the same meaning...)